src/output/OutputParser.cpp

```diff
@@ -0,0 +1,24 @@
+#include "OutputParser.hpp"
+
+#include <GraphMol/SmilesParse/SmartsWrite.h>
+#include <spdlog/spdlog.h>
+
+namespace coaler {
+    void OutputParser::add_aligned_mols(AlignedMolPair& aligned_mol_pair) {
```

Comment:
This thing does not parse stuff, but it writes stuff right? So `OutputWriter` would be better? Though it is implicit that output is written somehwere. So maybe `MolPairWriter` or smth similar?

Reply:
OutputWriter would be okay for me